src/main/resources/application-sqs.properties

```diff
@@ -0,0 +1,6 @@
+sqs.provider=aws
```

Comment:
would be better to just put this in application.yml as the defaults then you won't need this profile